docs/citing/index.rst

```diff
@@ -43,7 +43,7 @@ If you use OSQP for published work, we encourage you to put a star on `GitHub <h
           }
 
     GPU implementation
-        GPU implementation and PCG method for solving linear systems in this `paper <https://doi.org/10.1016/j.jpdc.2020.05.021>`__.
+        GPU implementation and PCG method for solving linear systems in this `paper <https://www.sciencedirect.com/science/article/pii/S0743731520303063>`__.
```

Comment:
I'd really prefer to keep the DOIs if at all possible. Do we know why the link checker is flagging the DOI-based links? Can it not handle the fact they are redirects?

Reply:
I agree - its just that the redirects were being reported by the link-checker, and I didn't pay close attention. I agree - the doi links are more canonical and what we should preserve anyway.